src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/CompareConverter.shared.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.CommunityToolkit.Extensions.Internals;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	/// <summary>
+	/// Converts an object that implements IComparable to an object or a boolean based on a comparaison.
+	/// </summary>
+	public class CompareConverter : CompareConverter<object>
```

Comment:
```suggestion
	public sealed class CompareConverter : CompareConverter<object>
```

Reply:
I was taking BoolToObjectConverter as a reference, so not sure if this needs to be sealed or no, if we change it we should do the same for [BoolToObjectConverter](https://github.com/xamarin/XamarinCommunityToolkit/blob/26c9dda6ef1ccc84a0f1fa53134a6888bc1da449/src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/BoolToObjectConverter.shared.cs#L11) also.